ledger_device_sdk/src/ecc.rs

```diff
@@ -523,6 +523,31 @@ impl SeedDerive for Ed25519 {
     }
 }
 
+/// Support SLIP10 derivation for Ed25519
+impl Ed25519 {
+    pub fn derive_from_path_slip10(path: &[u32]) -> ECPrivateKey<32, 'E'> {
+        let mut tmp = Secret::<64>::new();
+        let seed_key: &mut [u8; 12] = &mut [0; 12];
+        seed_key.copy_from_slice(b"ed25519 seed");
```

Comment:
This is the default value in the Os, so doesn't needed to be pass to os_perso_derive_node_with_seed_key